life/modernized/02-life-of-gilbert-latey.adoc

```diff
@@ -1704,9 +1759,12 @@ and compassion towards the said sufferers,
 for their lawful ease and relief from the present confinements,
 as in his wisdom and clemency shall seem most fitting and convenient.
 
+[.signed-section-context-close]
```

Comment:
Not sure if this should have been [sscc] or [sss]....I though it seemed a little better this way so that's what I went with.